catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopCatalogIT.java

```diff
@@ -151,6 +170,38 @@ private void dropSchema() {
     Assertions.assertFalse(catalog.asSchemas().schemaExists(schemaName));
   }
 
+  private String getFileInfos(NameIdentifier filesetIdent, String subPath, String locationName)
+      throws IOException {
+    String targetUrl =
+        "api/metalakes/"
+            + metalakeName
+            + "/catalogs/"
+            + catalogName
+            + "/schemas/"
+            + schemaName
+            + "/filesets/"
+            + filesetIdent.name()
+            + "/files";
+    Map<String, String> query = new HashMap<>();
+    if (!StringUtils.isBlank(subPath)) {
+      query.put("subPath", subPath);
+    }
+    if (!StringUtils.isBlank(locationName)) {
+      query.put("locationName", locationName);
+    }
+
+    FileInfoListResponse resp =
+        httpClient.get(
```

Comment:
Sorry, we can't use `HTTPClient` here. Instead, we should use `CloseableHttpClient` because we need to assert the raw response body string.